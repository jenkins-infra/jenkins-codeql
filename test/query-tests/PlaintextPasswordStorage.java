import hudson.util.Secret;

public class PlaintextPasswordStorage {
    private static String PASSWORD;

    private String password;

    private String credentialsId;

    private String prefixedCredentialsId;

    private String credentialId;

    private String prefixedCredentialId;

    private String credential;

    private String credentials;

    private Secret secretPassword;
}
