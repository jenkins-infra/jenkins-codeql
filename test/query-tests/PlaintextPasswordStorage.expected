| PlaintextPasswordStorage.java:6:20:6:27 | password | Field should be reviewed whether it stores a password and is serialized to disk: password |
| PlaintextPasswordStorage.java:16:20:16:29 | credential | Field should be reviewed whether it stores a password and is serialized to disk: credential |
| PlaintextPasswordStorage.java:18:20:18:30 | credentials | Field should be reviewed whether it stores a password and is serialized to disk: credentials |
